core/src/test/scala/integration/kafka/zk/ZkMigrationIntegrationTest.scala

```diff
@@ -1073,4 +1151,13 @@ class ZkMigrationIntegrationTest {
     kraftCluster.close()
     zkCluster.stop()
   }
+
+  def maybeRetry(shouldRetry: Boolean, maxWaitMs: Long)(block: => Unit): Unit = {
```

Comment:
It seems we don't set `shouldRetry` to `false` in this test. Maybe this method is unnecessary?